spec/factories/shopping_cart_items.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+ factory :shopping_cart_item do
+   quantity 1
+   price_cents 25
+   price_currency "SEK"
+ end
```

Comment:
I have be trying to read up on it and but i cant any info about it. but it seems to work without it.